### DISCUSSION TITLE
Symfony Messenger Retry

### DISCUSSION THREAD

**Discussion by gergobalogh** [symfony/symfony#58288] [Q&A]
Hi!
I have a question about the symfony messenger config of retries.
According to the documentation (https://symfony.com/doc/current/messenger.html#retries-failures), the retry settings are configurable in messenger.yaml. My question is:

In the context of AMQP transports, does the retry configuration apply to the process of dispatching a message to the transport (e.g., retrying if dispatch fails), or is it solely for retrying message consumption and handling (i.e., when an exception is thrown during message processing by a handler)?

In other words, does the retry strategy defined in messenger.yaml affect the behavior of dispatching messages to AMQP, or does it only apply when handling messages that are consumed from a transport?
